README-legacy.md

```diff
@@ -1,4 +1,7 @@
-noahong is a fork https://github.com/JDonner/NoAho
+# Legacy README
```

Comment:
I don't think we actually want to keep that in the long run?

Reply:
I did not want to cause a loss of information so I kept the original file and linked to it. We may want to amend/delete it later but it felt out of scope